packages/nestjs-logger/src/logger-transport.service.spec.ts

```diff
@@ -138,7 +139,9 @@ describe('LoggerTransportService', () => {
   });
 
   it('loggerService.logLevel_null', () => {
```

Comment:
just remove that tests that are checking for null if strict mode already handles this